{{cookiecutter.project_repo}}/Makefile

```diff
@@ -47,9 +47,14 @@ clean-test: ## remove test and coverage artifacts
 	rm -fr htmlcov/
 	rm -fr .pytest_cache
 
-lint: ## check style with flake8 and black
+lint: static-analysis ## check style with flake8 and black
 	flake8 {{ cookiecutter.project_slug }} tests setup.py
 	black --check --diff {{ cookiecutter.project_slug }} tests setup.py
+	mypy --ignore-missing-imports {{ cookiecutter.project_slug }} tests setup.py
+
+static-analysis: ## run static analysis and check licences of dependencies
+	bandit -r {{ cookiecutter.project_slug }}
+	liccheck -s liccheck.ini -r requirements.txt
```

Comment:
@elduddz Can you take a look please at bandit and liccheck please? The allowed licenses are in this PR in a file called `liccheck.ini`